### PR TITLE
Add About page

### PR DESCRIPTION
**Pull request by Winechicken**
close: #4
close: #6 

# Todo
- [x] separate about to components
- [x] move aut all base styles
CleanShot 2024-01-18 at 12 22 36@2x

- [x] header fro fixed to sticky
- [x] icons from png to svg
- [x] images replace with divs

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,18 +1,9 @@
 <script setup>
-	import { RouterLink, RouterView } from 'vue-router'
+
```
Comment (SavaCool122) [inline src/App.vue:2]: extra space, don't forget to use prettier

File: src/App.vue [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
-		<div class="wrapper">
-
-			<nav>
-				<RouterLink to="/">Home</RouterLink>
-			</nav>
-		</div>
-	</header>
 
 	<RouterView />
 </template>
```
Comment (SavaCool122) [inline src/App.vue:9]: forgot to delete styles after deleting tags

File: src/components/about/header/Header.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<!-- eslint-disable vue/multi-word-component-names -->
```
Comment (SavaCool122) [inline src/components/about/header/Header.vue:1]: lest move this rule from comment to `.eslintrc.cjs`

File: src/components/about/header/Header.vue [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+			<div>
+				<RouterLink to="/sign-in">Sign In</RouterLink>
+				<RouterLink to="/" class="mainLink">Go to Forms</RouterLink>
+			</div>
+		</nav>
+	</body>
+</template>
+
+<style>
+@import url('https://fonts.googleapis.com/css2?family=Red+Hat+Display&display=swap');
```
Comment (SavaCool122) [inline src/components/about/header/Header.vue:22]: let's not use Google font imports. These are unnecessary network requests when loading the page. it is better to download and connect them locally

File: src/components/about/header/Header.vue [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+	box-shadow: 0 2px 5px #9d9d9d;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	position: fixed;
+	top: 0;
+	padding: 20px;
+	font-family: 'Red Hat Display', sans-serif;
+}
+a {
```
Comment (SavaCool122) [inline src/components/about/header/Header.vue:35]: we can use modern css nesting 

```css
a {
	text-decoration: none;
	color: #121212;
	margin: 10px 40px;
	font-size: 20px;
	
	&:hover {
	  transition-duration: .4s;
	  color: #28297a;
	  text-shadow: 0 0 1px #28297a;
        }
}
```

and for .mainLink same

File: src/components/common/LinkButton.vue [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<script setup></script>
```
Comment (SavaCool122) [inline src/components/common/LinkButton.vue:1]: les's move mainLink from Header.vue to this component and reuse it in Header.vue

**Review by SavaCool122 — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (SavaCool122) [reply src/App.vue:2]: adasdasda

Comment (Winechicken) [reply src/App.vue:2]: Done, sorry, never happen again

File: src/App.vue (hunk 2 above)
Comment (Winechicken) [reply src/App.vue:9]: Done, sorry, never happen again

File: src/components/about/header/Header.vue (hunk 3 above)
Comment (Winechicken) [reply src/components/about/header/Header.vue:1]: Done, I'll delete the comments

File: src/components/about/header/Header.vue (hunk 4 above)
Comment (Winechicken) [reply src/components/about/header/Header.vue:22]: Ok, I'll manage it

File: src/components/about/header/Header.vue (hunk 5 above)
Comment (Winechicken) [reply src/components/about/header/Header.vue:35]: Done

File: src/components/common/LinkButton.vue (hunk 6 above)
Comment (Winechicken) [reply src/components/common/LinkButton.vue:1]: Done

File: src/App.vue [hunk 7]
```diff
@@ -59,17 +35,6 @@ nav a:first-of-type {
 		place-items: center;
 		padding-right: calc(var(--section-gap) / 2);
```
Comment (SavaCool122) [inline src/App.vue:36]: this css variable dont exist

**Review by SavaCool122 — CHANGES_REQUESTED** [review]

File: src/assets/tree1.png
Comment (SavaCool122) [inline src/assets/tree1.png:1]: please replace with svg

**Review by SavaCool122 — CHANGES_REQUESTED** [review]

File: src/components/about/header/Header.vue [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+<template>
+	<body>
+		<nav>
+			<div>
+				<a href="#overview">Overview</a>
+				<a href="#features">Features</a>
+			</div>
+			<div class="rightSide">
+				<RouterLink to="/sign-in">Sign In</RouterLink>
+				<LinkButton></LinkButton>
```
Comment (SavaCool122) [inline src/components/about/header/Header.vue:15]: please replace with single tag `<LinkButton></LinkButton>` with `<LinkButton />`

File: src/pages/HomePage.vue [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+<template>
+	<main>
+		<img src="../assets/logo.svg" alt="Vue.js logo" />
+		Какой-нибудь текст
+	</main>
+</template>
+
+<style>
+
+body {
```
Comment (SavaCool122) [inline src/pages/HomePage.vue:12]: lest move this code fragment to wrapper component

File: src/components/about/header/Header.vue [hunk 10]
```diff
@@ -0,0 +1,51 @@
+<script setup>
+import { RouterLink } from 'vue-router'
+import LinkButton from '../../common/LinkButton.vue'
+</script>
+
+<template>
+	<body>
```
Comment (SavaCool122) [inline src/components/about/header/Header.vue:7]: please remove boy from header

Comment (Winechicken) [reply src/components/about/header/Header.vue]: done

File: src/pages/HomePage.vue (hunk 9 above)
Comment (Winechicken) [reply src/pages/HomePage.vue]: done

File: src/components/about/header/Header.vue (hunk 8 above)
Comment (Winechicken) [reply src/components/about/header/Header.vue]: done and commited

File: src/assets/tree1.png
Comment (Winechicken) [reply src/assets/tree1.png:1]: all imgs are in svg now

**Review by SavaCool122 — APPROVED** [review]

**Pull request merged**